docs/apidiffs/current_vs_latest/opentelemetry-exporter-logging-otlp.txt

```diff
@@ -1,2 +1,5 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
+***! MODIFIED CLASS: PUBLIC FINAL io.opentelemetry.exporter.logging.otlp.OtlpJsonLoggingMetricExporter  (not serializable)
```

Comment:
eek. I'm guessing we messed up here and our metrics exporter ended up in an otherwise stable module

Reply:
Ah, I was kinda curious this showed up, I figured since it was the logging exporter that wasn't stable yet.  I'll re-add and deprecate.